src/App.tsx

```diff
@@ -1,10 +1,52 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { showError } from './helpers/helpers';
+import { Error } from './components/Error';
+import { Footer } from './components/Footer';
+import { Todos } from './components/Todos';
+import { Header } from './components/Header';
 
-const USER_ID = 0;
+const USER_ID = 10881;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [filter, setFilter] = useState('all');
+  const [error, setError] = useState('');
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => (showError('Failed to get list of todos', setError)));
```

Comment:
unnecessary helper

Reply:
I will use it for next tasks. I can write SetErorr and and setTimeout for every error but i think will be beter  use this function, because we have similar cases, just different text.
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/130560419/31415dc7-7358-402b-aaaa-7bc2019c048a)